src/main/java/school/faang/first_stream/ListOperations.java

```diff
@@ -0,0 +1,49 @@
+package school.faang.first_stream;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.function.Predicate;
+
+public class ListOperations {
```

Comment:
в аждом методе для читаемости стоит переносить каждую операцию стрима на следующую строку вместе с точкой:
```suggestion
list.stream()
     .map()
     .filter()
```